internal/stackdriver/stackdriver.go

```diff
@@ -12,19 +12,21 @@ import (
 
 	// TODO: Migrate to cloud.google.com/go/monitoring/apiv3/v2 once RPC for MQL
 	// query is added (https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.timeSeries/query).
-
 	monitoring "google.golang.org/api/monitoring/v3"
 )
 
+// query is the filter used to retrieve metrics data.
+type query struct {
+	filter string
+}
+
```

Comment:
can you `type query string`

Reply:
Done! Can you confirm if the best way to pass a string to the Cloud Monitoring package is type conversion `string(query)`?